### PR TITLE
make new devel branch

### PR DESCRIPTION
**Pull request by kwmonroe**
Now that `master` represents the latest stable jujubox, merge it into our devel branch.  Then add `ppa:juju/devel` so jujubox:devel will install the latest devel version of juju.

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -1,35 +1,50 @@
...
-apt-get install -qy software-properties-common
-apt-add-repository -y ppa:juju/devel
+# Install software tools such as add-apt-repository.
+apt-get install -y software-properties-common
+# devel ppa is required for juju2 beta release
+apt-add-repository -u -y ppa:juju/devel
 # stable ppa is required for charm-tools
-apt-add-repository -y ppa:juju/stable
-apt-get update -qq
+apt-add-repository -u -y ppa:juju/stable
```
Comment (mbruzek) [inline setup.sh]: We may be able to remove the stable from here if you only want devel juju

Comment (kwmonroe) [reply setup.sh]: i was going off the comment from `master` that said stable was required for charm-tools.  It appears that the stable ppa has a more recent version of c-t (2.1.9):

https://launchpad.net/~juju/+archive/ubuntu/stable

Versus devel's version of c-t (2.1.2):

https://launchpad.net/~juju/+archive/ubuntu/devel

Unless you know better, I think we want stable in there to pick up the most current charm-tools.

Comment (lazypower) [reply setup.sh]: That can be added in the downstream box image that requires this. It makes more sense to encapsulate that there rather htan here in a lower layer, where its inclusion is effectively hidden unless there are other reasons for having both PPA's enabled in jujubox.

Comment (kwmonroe) [reply setup.sh]: Ahhh, i see.  charm-tools is no longer installed in jujubox, and downstream charmbox installs it from git with pip.

I'll remove the stable ppa and see if that breaks anything else from setup.sh and charmbox.

**Comment by kwmonroe** [comment]
Tested local `jujubox:devel` and `charmbox:devel` after removing `ppa:juju/stable`.  Both look good.

**Comment by lazypower** [comment]
This sounds and looks good to me if its +1 by @mbruzek. Cursory glance and build was LGTM

**Comment by mbruzek** [comment]
Yeah this now looks like something we can use.

**Pull request merged**